python/python/bystro/proteomics/default_query_fields.py

```diff
@@ -0,0 +1,182 @@
+"""Default fields to pass to OpenSearch query."""
+
+DEFAULT_FIELDS = [
```

Comment:
Do we need this still?

Reply:
Let's drop Default Fields. We'll need a build step that generates this automatically from the yaml configs.